docs/howto/metrics/proc_exposing-application-metrics.adoc

```diff
@@ -37,7 +37,7 @@ include::prometheus.yml[tag=thorntail]
 ----
 +
 The default behavior of {Thorntail}-based applications is to expose metrics at the `/metrics` endpoint.
-This is what the MicroProfile Metrics specification requires, and also what Prometheus expects.
+This is what the Eclipse MicroProfile Metrics specification requires, and also what Prometheus expects.
```

Comment:
This is part of the "Metrics" guide. Could we make sure the guide uses the full name somewhere at the beginning, and then leave the "Eclipse" word out? It reads worse now.